router/base-http/src/main/scala/io/buoyant/router/http/AddForwardedHeader.scala

```diff
@@ -42,13 +43,35 @@ class AddForwardedHeader(byLabel: () => String, forLabel: () => String) extends
   }
 }
 
-object AddForwardedHeader {
```

Comment:
I think rather than move this into http-base, we should split it into two: one in router/http and one in router/h2

Reply:
possibly with common functionality factored into a trait or util object that can stay in http-base